mergify_engine/web/__init__.py

```diff
@@ -274,26 +275,20 @@ async def subscription_cache_update(
     return responses.Response("Cache updated", status_code=200)
 
 
+async def cleanup_subscription(owner_id: int) -> None:
+    await subscription.Subscription.delete(_AREDIS_CACHE, owner_id)
+    await user_tokens.UserTokens.delete(_AREDIS_CACHE, owner_id)
+
+
 @app.delete(
     "/subscription-cache/{owner_id}",  # noqa: FS003
     dependencies=[fastapi.Depends(auth.signature)],
 )
 async def subscription_cache_delete(owner_id):  # pragma: no cover
-    await _AREDIS_CACHE.delete(f"subscription-cache-owner-{owner_id}")
+    await cleanup_subscription(owner_id)
```

Comment:
There's no need to clean up the token cache here.

Actually, we should delete the token from the engine cache when we update the account on the dashboard and it becomes invalid, but we don't do it right now. If we add a new endpoint to the engine to delete cached token, we could improve the current workflow.